heka/sandbox/filters/telemetry_decoder_view.lua

```diff
@@ -106,5 +119,13 @@ function timer_event(ns)
         last_cleared = ns
     end
 
+    if not cb:get(ns, 1) then
+        cb:add(ns, 1, 0/0) -- always advance the buffer/graph
```

Comment:
Can you update the comment to `always advance the buffer/graph, defaulting to NaN`?